### PR TITLE
Scaffold out the frontend tools for dev

### PR DESCRIPTION
**Pull request by earnubs**
Babel.
Webpack, hot, React.
Eslint.
Jest, enzyme, snapshots.

### CODE REVIEW

File: frontend/src/LazyInvoice.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { hot } from 'react-hot-loader/root';
+import React from 'react';
+
+export const Component = () => <div>Hello Fabien!</div>;
```
Comment (fgallina) [inline frontend/src/LazyInvoice.js]: /Fabien/Feibian/

File: frontend/webpack.dev.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const path = require('path');
```
Comment (fgallina) [inline frontend/webpack.dev.js]: I'm not sure I totally understand the need of these two split webpack configs, I see some differences here and there but in the end both seem to target development. I see one enables `HotModuleReplacementPlugin`, is that so we have two ways of running things? if so I'm all good with that but I'd document these files a bit.

Comment (earnubs) [reply frontend/webpack.dev.js]: My bad, there's some accidental file adding going on here ...

File: frontend/src/LazyInvoice.js (hunk 1 above)
Comment (earnubs) [reply frontend/src/LazyInvoice.js]:
```suggestion
export const Component = () => <div>Hello Feibian!</div>;
```

**Pull request merged**